ansible/pbTestScripts/testScript.sh

```diff
@@ -184,6 +188,12 @@ startVMPlaybookWin()
 		# Add the "ansible_winrm_transport" to adoptopenjdk_variables.yml
 		echo -e "\nansible_winrm_transport: credssp" >> playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml
 	fi
+        # If the disk size is less than 970MB, resize the C: drive to be as big as possible
+	local sizeOfDisk=$(du -k $WORKSPACE/.vagrant.d/boxes/mwrock*/*/virtualbox/*.vmdk | cut -f 1)
+        if [ $sizeOfDisk -lt 9700000 ]; then
```

Comment:
Is there an extra zero in here?

Reply:
Indeed there is, however, I've noticed that that file size doesn't change even once the volume is extended... I'm not really sure why, so I'm going to find a different way to do that `if` statement